src/components/style/interactive/button.css

```diff
@@ -74,71 +74,128 @@
-  background-color: #1C6695;
+  background-color: var(--color-blue);
+}
+
+.fdsButton--blue.fdsButton--isActive {
+  background-color: #5297b9;
 }
 
 .fdsButton--blue:hover {
-  background-color: #29719D;
+  background-color: #1472a1;
 }
 
 .fdsButton--blue:active {
-  background-color: #5C96B6;
+  background-color: #5297b9;
+}
+
+.fdsButton--blue.fdsButton--isActive:focus {
+  background-color: #5297b9;
 }
 
 .fdsButton--blue.fdsButton--isDestructive {
-  background-color: var(--button-destructive);
+  background-color: var(--color-red);
+}
+
+/* stylelint-disable-next-line selector-max-class  */
+.fdsButton--blue.fdsButton--isDestructive.fdsButton--isActive {
+  background-color: #ed737c;
 }
 
 .fdsButton--blue.fdsButton--isDestructive:hover {
-  background-color: var(--button-destructive-hover);
+  background-color: #e7424d;
 }
 
 .fdsButton--blue.fdsButton--isDestructive:active {
-  background-color: var(--button-destructive-pressed);
+  background-color: #ed737c;
 }
 
-.fdsButton--outlined {
-  border: 1px solid var(--button-outlined-border);
-  background-color: white;
-  color: var(--font-color-primary);
+/* stylelint-disable-next-line selector-max-class  */
+.fdsButton--blue.fdsButton--isDestructive.fdsButton--isActive:focus {
+  background-color: #ed737c;
 }
 
+.fdsButton--outlined,
 .fdsButton--ghost {
-  /* this isn't needed, just writing out for clarity */
-  color: var(--button-primary-color);
+  color: var(--color--charcoal);
 }
 
-.fdsButton--outlined:hover {
-  background-color: #F2F2F2;
+.fdsButton--outlined {
+  border: 1px solid #c4c4c4;
+  background-color: var(--color-white);
 }
 
-.fdsButton--ghost:hover {
-  background-color: var(--button-secondary-hover);
-  color: var(--button-secondary-hover-color);
+.fdsButton--outlined.fdsButton--isActive {
+  background-color: #d9d9d9;
+}
+
+.fdsButton--outlined:hover {
+  background-color: #f0f0f0;
 }
 
 .fdsButton--outlined:active {
-  background-color: #E0E0E0;
+  background-color: #d9d9d9;
+}
+
+.fdsButton--outlined.fdsButton--isActive:focus {
+  background-color: #d9d9d9;
+}
+
+.fdsButton--ghost.fdsButton--isActive {
+  background-color: rgba(64, 64, 64, 0.2);
+}
+
+.fdsButton--ghost:hover {
+  background-color: rgba(64, 64, 64, 0.08);
 }
 
 .fdsButton--ghost:active {
-  background-color: var(--button-secondary-pressed);
-  color: var(--button-secondary-pressed-color);
+  background-color: rgba(64, 64, 64, 0.2);
+}
+
+.fdsButton--ghost.fdsButton--isActive:focus {
+  background-color: rgba(64, 64, 64, 0.2);
+}
+
+/* stylelint-disable-next-line selector-max-class  */
+.fdsButton--outlined.fdsButton--isDestructive.fdsButton--isActive {
+  background-color: #fad6d8;
+}
+
+.fdsButton--outlined.fdsButton--isDestructive:hover {
+  background-color: #fdeff0;
+}
+
+.fdsButton--outlined.fdsButton--isDestructive:active {
+  background-color: #fad6d8;
+}
+
+/* stylelint-disable-next-line selector-max-class  */
+.fdsButton--outlined.fdsButton--isDestructive.fdsButton--isActive:focus {
```

Comment:
You may wonder why in this and other places I'm using such freakishly long selectors 😅 it's probably too abstract to explain here, but happy to give a live demo.

Reply:
i think I see what's going on here. If we have to keep adding these combos, we could roll that up into its own modifier (e.g. `fdsButton--activeDestructiveOutlined`). I don't see a need to accommodate that for now though.

Reply:
That's actually a good point but there's something else here.

So the order of precedence is as follows:

isActive,
hover,
:active

The problem with that, is when a user activates a button, it won't look active, as the user is still hovered over it.

isActive,
hover,
:active
isActive:focus

What I've done, is utilize :focus so that if the user is activating a button, the button will have focus, and I use that to give precedence to the isActive state again.